packages/vite-plugin-cloudflare/src/constants.ts

```diff
@@ -4,3 +4,5 @@ export const ASSET_WORKERS_COMPATIBILITY_DATE = "2024-10-04";
 // TODO: add `Text` and `Data` types (resolves https://github.com/cloudflare/workers-sdk/issues/8022)
 export const MODULE_TYPES = ["CompiledWasm"] as const;
 export type ModuleType = (typeof MODULE_TYPES)[number];
+
+export const DEFAULT_INSPECTOR_PORT = 9229;
```

Comment:
Note: this is another instance in which a shared package in the monorepo would make sense to me, so that all packages could share the same constant for the default inspector